### PR TITLE
Pinfo

### PR DESCRIPTION
**Pull request by ukscid**
Bugfix for splitnames
"lname, fname" results in a leading blank to " fname .."

chess24.com has moved to chess.com and a real good player webside.
resolvers for chessbase and chess.com added

### CODE REVIEW

File: tcl/tools/pinfo.tcl [hunk 1]
```diff
@@ -189,6 +215,8 @@ proc ::pinfo::splitName { playerName } {
   set count [string first "," $playerName ]
   if { $count < 0 } {
     set count [string first " " $playerName ]
+  } elseif { [string index $playerName [expr $count + 1]] eq " " } {
```
Comment (benini) [inline tcl/tools/pinfo.tcl]: What are you trying to do?

**Comment by ukscid** [comment]
Input "Lastname, Firstname" results in  list: " Lastname" "Firstname". There is a blank at the start of lastname.
This leads to a url like https://chess.com/en/players/%%20firstname-lastname or
https://players.chessbase.com/de/player/lastname_%%20firstname
Most website ignore this additional blank (%%20) and work correct, but chess.com does not and throws a 404 Page not found error.
This fix should remove this blank before the lastname.

**Comment by benini** [comment]
Do you just want to trim whitespaces?

**Comment by ukscid** [comment]
You are right. I changed it to use [string trim ..]

**Pull request merged**